src/CookieJar/PersistentCookieJar.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Http\Message\CookieJar;
+
+use Http\Message\CookieJar;
+
+/**
+ * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
+ */
+interface PersistentCookieJar extends CookieJar
```

Comment:
is this our job or should that be handled with instantiation? it seems a bit out of scope here to define this.